### DISCUSSION TITLE
dark mode web page and changes to raito

### DISCUSSION THREAD

**Discussion by mckaygerhard** [BashGui/easybashgui#32] [Ideas]
as pointed https://github.com/BashGui/easybashgui/issues/30 by @zen0bit

**Comment by mckaygerhard** [comment]
hey @zen0bit  any news on dark mode? can you put a pull request? i see your changes in the repo from your fork.. can you post a PR?

**Comment by zen0bit** [comment]
Need more investigation. Don't know how raito works internally (what needs to be changed in index.html)

I can create PR but nothing helpful there yet except rising theme in that one line..

**Reply by mckaygerhard** [reply]
raito is just a javascript logic around the markdown javacrip library.. nothing more.. 

it read the files requested by browser and change the nodes by injection of html.. 

i thinks you should talk with author of raito! @zen0bit

**Reply by zen0bit** [reply]
sure but i need learn bit more about raito first, before bothering maintaner

**Comment by zen0bit** [comment]
Not sure difference between

```
:function {
	...
}
```

```
#function {
	...
}
```

```
function {
	...
}
```

you introduced with raito. Whats commented out, are they functions (know only bit of bash) why is :root { ... } ??? whats all of that?

	:root {
	    --text-color: #000000;
	    --text-color: #000000;
	    --border-color: #a7cefc;
	    --border-color: #a7cefc;
	    --border-h-color: #313ae2;
	    --border-h-color: #313ae2;
	    --border-hl-color: #dbdbdb;
	    --border-hl-color: #dbdbdb;
	    --background-color: white;
	    --background-color: white;
	    --background-navbar-color: #0059c5;
	    --background-navbar-color: #0059c5;
	    --darker-background-color: #d1d7df2f;
	    --darker-background-color: #d1d7df2f;
	    --lighter-background-color: #fbfdff;
	    --lighter-background-color: #fbfdff;
	}

	#docs-navbar {
	    margin: 0;
	    margin: 0;
	    overflow: hidden;
	    overflow: hidden;
	    background-color: var(--background-navbar-color);
	    background-color: var(--background-navbar-color);
	    position: fixed;
	    position: fixed;
	    width: 100%;
	    width: 100%;
	    padding-left: 2rem;
	    padding-left: 2rem;
	    z-index: 5;
	    z-index: 5;
	}

**Comment by mckaygerhard** [comment]
> you introduced with raito. Whats commented out, are they functions (know only bit of bash) why is :root { ... } ??? whats all of that?

there are two part of the code that bootstrap and javascrip dont have inherence, the navbar logic (components) and the rendering logic (markdown.js), 

that part of the code it put the navbar in the top (with all the logic, by example over all the layers), and apply the CSS from boostwatch.. also render the markdown .. 

@vaisarger  and @zen0bit  in fact i dont like javascript but the development of raito does  not implict stupid crap tools with python neither generation of compresed css/js files.. so its so perfect! for me! javascript is overstimated and too much used ..

**Reply by zen0bit** [reply]
I mean why is somewhere `#` somwhere `:` and somewhere nothing before [name of function?] {
}
